core/app/jobs/spree/gift_cards/bulk_generate_job.rb

```diff
@@ -0,0 +1,25 @@
+module Spree
+  module GiftCards
+    class BulkGenerateJob < ::Spree::BaseJob
+      queue_as Spree.queues.gift_cards_bulk_generate
```

Comment:
```suggestion
      queue_as Spree.queues.gift_cards
```

also please add this new queue to https://github.com/spree/spree/blob/75960e1609e094fe1add1f951289a7bea095f7c2/core/lib/spree/core.rb#L97